### PR TITLE
PODS-3766: Added member details view and form and model, tweaked view…

### PR DESCRIPTION
**Pull request by vinnicombe**
…s to use govukButton instead of hard coded links

### CODE REVIEW

File: app/pages/MemberDetailsPage.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+
+package pages
+
+import models.MemberDetails
+import play.api.libs.json.JsPath
+
+case object MemberDetailsPage extends QuestionPage[MemberDetails] {
+
```
Comment (madhuranakate) [inline app/pages/MemberDetailsPage.scala]: needs to take an index

File: conf/app.routes [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
-GET        /:srn/new-return/change/charge-type                                   controllers.ChargeTypeController.onPageLoad(mode: Mode = CheckMode, srn: String)
-POST       /:srn/new-return/change/charge-type                                   controllers.ChargeTypeController.onSubmit(mode: Mode = CheckMode, srn: String)
+GET         /:srn/new-return/charge-type                                               controllers.ChargeTypeController.onPageLoad(mode: Mode = NormalMode, srn: String)
+POST        /:srn/new-return/charge-type                                               controllers.ChargeTypeController.onSubmit(mode: Mode = NormalMode, srn: String)
+GET         /:srn/new-return/change/charge-type                                        controllers.ChargeTypeController.onPageLoad(mode: Mode = CheckMode, srn: String)
+POST        /:srn/new-return/change/charge-type                                        controllers.ChargeTypeController.onSubmit(mode: Mode = CheckMode, srn: String)
+
+GET         /:srn/new-return/annual-allowance-charge/members-details                   controllers.chargeE.MemberDetailsController.onPageLoad(mode: Mode = NormalMode, srn: String)
```
Comment (madhuranakate) [inline conf/app.routes]: these urls need an index as third parameter

File: conf/views/chargeE/annualAllowanceYear.njk [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                isPageHeading: true,
+                classes: "govuk-fieldset__legend--xl"
+              }
+            },
+            items: radios,
+            errorMessage: form.value.error
+          }) }}
+
+          {{ govukButton({
+            text: messages("site.continue")
```
Comment (vinnicombe) [inline conf/views/chargeE/annualAllowanceYear.njk]: Add id via attributes object, see chargeType.njk

File: conf/views/chargeE/annualAllowanceYear.njk [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          }) }}
+
+        </form>
+
+      </div>
+    </div>
+  </div>
+
+  <a href= {{ returnUrl }} class="govuk-link"> {{ messages("return.to.link", viewModel.schemeName) }}</a>
```
Comment (vinnicombe) [inline conf/views/chargeE/annualAllowanceYear.njk]: `href= {{ returnUrl }}` needs to be `href="{{ viewModel.returnUrl }}"` (we're making this mistake all over the place)

File: conf/views/chargeE/whatYouWillNeed.njk [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+          <li>{{ messages("chargeE.whatYouWillNeed.li4") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li5") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li6") }}</li>
+        </ul>
+
+        <a href = {{nextPage}} class="govuk-button">{{ messages("site.continue") }}</a>
+      </div>
+    </div>
+
+  <a href = nextPage class="govuk-link"> {{ messages("return.to.link", schemeName) }}</a>
```
Comment (vinnicombe) [inline conf/views/chargeE/whatYouWillNeed.njk]: `href = nextPage` will need to be `href="{{nextPage}}"`, one above will need quotes added too.

File: conf/views/chargeE/annualAllowanceYear.njk [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+                      attributes: {id: "submit"}
+             }) }}
+
+        </form>
+
+      </div>
+    </div>
+  </div>
+
+  <a href= ""{{ viewModel.returnUrl }}"" class="govuk-link"> {{ messages("return.to.link", viewModel.schemeName) }}</a>
```
Comment (vinnicombe) [inline conf/views/chargeE/annualAllowanceYear.njk]: Just single quotes

File: conf/views/chargeE/whatYouWillNeed.njk [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+          <li>{{ messages("chargeE.whatYouWillNeed.li1") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li2") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li3") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li4") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li5") }}</li>
+          <li>{{ messages("chargeE.whatYouWillNeed.li6") }}</li>
+        </ul>
+
+        <a href = {{nextPage}} class="govuk-button">{{ messages("site.continue") }}</a>
+      </div>
```
Comment (vinnicombe) [inline conf/views/chargeE/whatYouWillNeed.njk]: Quotes missing here

File: conf/views/chargeE/whatYouWillNeed.njk [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    </div>
+
+  <a href = "{{nextPage}}" class="govuk-link"> {{ messages("return.to.link", schemeName) }}</a>
```
Comment (vinnicombe) [inline conf/views/chargeE/whatYouWillNeed.njk]: Extra space before quote

**Pull request merged**